monitoring/sysctl_checkers.go

```diff
@@ -53,11 +53,63 @@ func NewMayDetachMountsChecker() *SysctlChecker {
 	}
 }
 
+// NewCNIForwardingChecker checks if CNI interface has forwarding enabled
+func NewCNIForwardingChecker() *SysctlChecker {
+	return &SysctlChecker{
+		CheckerName:     CNIForwardChecker,
+		Param:           "net.ipv4.conf.cni0.forwarding",
+		Expected:        "1",
+		OnValueMismatch: "ipv4 forwarding is off on interface cni0, see https://www.gravitational.com/docs/faq/#ipv4-forwarding",
+		SkipNotFound:    true, // interface may not exist, so skip if not found
+	}
+}
+
+// NewFlannelForwardingChecker checks if flannel interface has forwarding enabled
+func NewFlannelForwardingChecker() *SysctlChecker {
+	return &SysctlChecker{
+		CheckerName:     FlannelForwardChecker,
+		Param:           "net.ipv4.conf.flannel/1.forwarding",
+		Expected:        "1",
+		OnValueMismatch: "ipv4 forwarding is off on interface flannel.1, see https://www.gravitational.com/docs/faq/#ipv4-forwarding",
+		SkipNotFound:    true, // interface may not exist, so skip if not found
+	}
+}
+
+// NewWormholeBridgeForwardingChecker checks if wormhole-br0 interface has forwarding enabled
+func NewWormholeBridgeForwardingChecker() *SysctlChecker {
+	return &SysctlChecker{
+		CheckerName:     WormholeBridgeForwardChecker,
+		Param:           "net.ipv4.conf.wormhole-br0.forwarding",
```

Comment:
Are all of these interfaces always guaranteed to be present in the cluster? If any of them are missing for some reason, their corresponding checks will be failing?

Reply:
I don't think so - that's what I assume `SkipNotFound` is for.

Reply:
Oh lol. Sorry I looked at this before my morning coffee :man_shrugging: